sdformat_to_mjcf/tests/test_add_geometry.py

```diff
@@ -22,9 +22,8 @@
 from ignition.math import Pose3d, Vector2d, Vector3d
 from dm_control import mjcf
 
-from tests import helpers
-from sdformat_mjcf.converters import geometry as geometry_conv
-
+import helpers
```

Comment:
Last time I checked, we needed to import `helpers` from `tests`.
```suggestion
from tests import helpers
```